### PR TITLE
Try a second icon_dir location.

### PR DESCRIPTION
**Pull request by ralphbean**
I just hit this when rebuilding the prod index.

### CODE REVIEW

File: bin/fcomm-index-packages [hunk 1]
```diff
@@ -75,6 +75,8 @@ if __name__=="__main__":
             log.warn("No icons_dest specified.  Skipping copy.")
         else:
             icon_dir = os.path.join(options.cache_path, 'icons')
+            if not os.path.isdir(icon_dir):
+                icon_dir = os.path.join(options.cache_path, '..', 'icons')
```
Comment (pypingou) [inline bin/fcomm-index-packages]: Should we also check if this one is a directory?

**Comment by pypingou** [comment]
One question, otherwise looks fine

File: bin/fcomm-index-packages (hunk 1 above)
Comment (ralphbean) [inline bin/fcomm-index-packages]: Heh, we could.. but it fails on the next line anyways (which is what prompted me to add this..)

Comment (pypingou) [inline bin/fcomm-index-packages]: wfm then :)

**Pull request merged**